modules/http4s/src/test/scala/io/chrisdavenport/fuuid/http4s/FUUIDVarSpec.scala

```diff
@@ -0,0 +1,31 @@
+package io.chrisdavenport.fuuid.http4s
+
+import io.chrisdavenport.fuuid.{FUUID, FUUIDArbitraries}
+import org.http4s.dsl.io._
+import org.scalacheck._
+import org.specs2.ScalaCheck
+import org.specs2.mutable.Specification
+
+class FUUIDVarSpec extends Specification with ScalaCheck with FUUIDArbitraries {
+
+  "FUUID Extractor in Path" should {
+
+    "work properly given a valid UUID" in prop { validFuuid: FUUID =>
+
+      (Path(s"/v1/${validFuuid.toString}") match {
```

Comment:
Could we use `show` interpolator here for best practice demonstration.

Reply:
:+1: but #25 is required to make it work